cosmosdb/management_endpoint.bal

```diff
@@ -443,15 +452,15 @@ public client class ManagementClient {
         return mapJsonToTrigger(jsonResponse);
     }
 
-    # Replace an existing trigger.
+    # Replaces an existing trigger.
     #
     # + databaseId - ID of the database to which the container belongs to
     # + containerId - ID of the container which contains the trigger
     # + triggerId - The ID of the trigger to be replaced
     # + trigger - A JavaScript function represented as a string
     # + triggerOperation - The specific operation in which trigger will be executed
     # + triggerType - The instance in which trigger will be executed `Pre` or `Post`
```

Comment:
If this value is selected by the user, can we have the description in the following format?

"Select `Pre` to execute the trigger before XXX, or `Post` to execute it after XXX"